python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py

```diff
@@ -0,0 +1,312 @@
+            self.monochromator.start_task,
+        )
+
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/SlewTelescopeIcrs.yaml
+            title: PowerOnATCalSys v1
+            description: Configuration for PowerOnATCalSys.
+              Each attribute can be specified as a scalar or array.
+              All arrays must have the same length (one item per image).
+            type: object
+            properties:
+              chiller_temperature:
+                description: Set temperature for the chiller
+                type: number
+                default: 20
+                minimum: 15
+
+              whitelight_power:
+                description: White light power.
+                type: number
+                default: 910
+                minimum: 0
+
+              wavelength:
+                description: Wavelength (nm). 0 nm is for white light.
+                type: number
+                default: 0
+                minimum: 0
+
+              grating_type:
+                description: Grating type for each image. The choices are 0=blue, 1=red, 2=mirror.
+                type: integer
+                enum: [0, 1, 2]
+                default: 2
+
+              entrance_slit_width:
+                description: Width of the monochrometer entrance slit (mm)
+                type: number
+                minimum: 0
+                default: 7
+
+              exit_slit_width:
+                description: Width of the monochromator entrance slit (mm)
+                type: number
+                minimum: 0
+                default: 7
+
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config : ``self.cmd_configure.DataType``
+
+        Raises
+        ------
+        salobj.ExpectedError :
+            If the data does not match the schema, or the lengths
+            of all values that are arrays do not match.
+        """
+        self.log.info("Configure started")
+
+        self.chiller_temperature = config.chiller_temperature
+        self.whitelight_power = config.whitelight_power
+        self.wavelength = config.wavelength
+        self.grating_type = config.grating_type
+        self.entrance_slit_width = config.entrance_slit_width
+        self.exit_slit_width = config.exit_slit_width
+
+        # if self.white_light_source is None:
+        #     self.white_light_source = salobj.Remote(
+        #         domain=self.domain, name="ATWhiteLight"
+        #     )
+
+        # if self.monochromator is None:
+        #     self.monochromator = salobj.Remote(
+        #         domain=self.domain, name="ATMonochromator"
+        #     )
+
+        self.config = config
```

Comment:
Above you are creating individual attributes to get the values from `config` and then here you just store the entire thing. I think pick one method. Either set them up individually or get the full thing, but not both.

Reply:
Thanks!!